java/src/jmri/jmrit/throttle/ThrottleWindow.java

```diff
@@ -755,6 +736,20 @@ public synchronized void addPropertyChangeListener(java.beans.PropertyChangeList
     public synchronized void removePropertyChangeListener(java.beans.PropertyChangeListener l) {
         pcs.removePropertyChangeListener(l);
     }
+    
+    private void installInputsListenerOnAllComponents(Container c) {
+        c.addKeyListener(myInputsListener);
+        c.addMouseWheelListener(myInputsListener);        
+        Component[] components = c.getComponents();
+        for (Component component : components) {
+            if (component instanceof Container) {
+                installInputsListenerOnAllComponents( (Container) component);
+            } else {
+                component.addKeyListener(myInputsListener);
+                component.addMouseWheelListener(myInputsListener);                
+            }
+        }
+    }    
 
-    private final static Logger log = LoggerFactory.getLogger(ThrottleWindow.class);
+    private final static Logger LOG = LoggerFactory.getLogger(ThrottleWindow.class);
```

Comment:
I'm aware that the Java standard is to have constants as upper case. But in JMRI, almost all `log` instances are lower case, so I think it would be good to keep it lower case here too.

Reply:
done